### DISCUSSION TITLE
2021-12-08 Weekly Meeting 33 Agenda / Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#139] [Weekly Meetings]
Weekly Meeting 33 is happening on Wednesday December 8th, 2021 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda

   - Welcome new members and attendance
   - Backlog refinement
     - Github Milestones is a much better way of swarming around sub projects (brief demo)
     - VOTE: Forming working group for OMI-Chan avatar https://github.com/omigroup/omigroup/issues/137
     - VOTE: A feedback channel would help me keep our backlog orderly, and newcomers can ask and answer questions. Do w have any issue with creating a new feedback channel as omigroup/feedback? https://github.com/omigroup/omigroup/issues/129
     - Per last week, did we form a working group for the Partner Program effort? https://github.com/omigroup/omigroup/issues/121
     - https://github.com/omigroup/omigroup/issues/109
     - [website](https://github.com/omigroup/website/projects/1) updates 
     - https://github.com/omigroup/omigroup/issues/136
   - Open metaverse community calendar: #130 
   - Proposed WASM UGC Scripting Working Group: #133
   - [glTF subgroup](https://github.com/omigroup/gltf-extensions/discussions/31)
     - Khronos Liaison update? 
  
 - Questions / Decisions:    
   - [What does the fiscal development group need?](https://github.com/omigroup/omigroup/discussions/67#discussioncomment-1719482) - does the core omigroup repo work for managing this or do we want a repo for the working group? See notes. 
   - Do we have any new CFP submissions? 
     - https://github.com/omigroup/omigroup/issues/134

 ## Add Ons
   - Upcoming Events: 
     - [VKet Booth ](https://github.com/omigroup/omigroup/issues/89)
     - WebXR Hackathon - "Immersive Web Fest"
     - [Avacon](https://www.avacon.org/blog/)
  - Reminder Twitter Spaces with Evo / Jin
    - others?

**Feel free to add items in the comments below** :point_down:

**Comment by mrmetaverse** [comment]
Do we want a new Wednesday meeting structure? https://github.com/omigroup/omigroup/discussions/140

**Comment by mrmetaverse** [comment]
Attendance

![image](https://user-images.githubusercontent.com/63426722/145251494-f6d722e3-6ea6-492f-acd2-b51fe34ce90d.png)

**Reply by robertlong** [reply]
![image](https://user-images.githubusercontent.com/1753624/145251703-6606d403-a857-46f0-96d2-626e71177ddd.png)

**Comment by mrmetaverse** [comment]
## Partner Program Working Group
Opened this ticket to form the agreed upon group for exploring the OMI partner program: https://github.com/omigroup/omigroup/issues/143
Closed this ticket as we voted and approved last week: https://github.com/omigroup/omigroup/issues/121

**Comment by mrmetaverse** [comment]
## OMI-Chan working group 

opened: https://github.com/omigroup/omigroup/issues/144
closed: https://github.com/omigroup/omigroup/issues/137

**Comment by mrmetaverse** [comment]
## Mapping the Metaverse 

- team to get better about making announcements 
- work tracked [here](https://github.com/omigroup/omigroup/issues/109)
- there is a Miro floating around somewhere 
- there are more events coming up 
- need to add to the #130 :calendar:

**Reply by mrmetaverse** [reply]
have been using twitter spaces, and will most likely use through the end of december. But to speak you must have the app.

**Reply by mrmetaverse** [reply]
New projects board for managing the backlog: https://github.com/omigroup/omigroup/projects/3

**Comment by mrmetaverse** [comment]
## WASM Working Group 

Voted to approve the working group and closed this ticket: https://github.com/omigroup/omigroup/issues/133
Created this ticket to form the group: https://github.com/omigroup/omigroup/issues/146

**Comment by mrmetaverse** [comment]
## VKet Booth 

Koolala and the V-Sekai team really hooked up OMI with a VKET booth in Virtual Market (VR Chat) 
You can find it in the [Vket2021 Poppin' Jump - Rich Gold](https://vrchat.com/home/world/wrld_4f73aad2-6887-4f85-9fb7-7c075562b849) Server in the "Alpha" Room. #89

**Comment by mrmetaverse** [comment]
## Website Working group 

- https://github.com/omigroup/website/projects/1
- @jsejms recently added comments and changes to the Style Guide 
- @robertlong started making changes to the website last week and will resume in a couple of weeks when things have calmed down. 
- If you'd like to support this effort please contact @mrmetaverse or @Elirudite
